docs/source/markdown/options/add-host.md

```diff
@@ -1,7 +1,7 @@
 ####> This option file is used in:
 ####>   podman build, create, pod create, run
 ####> If you edit this file, make sure your changes
-####> are applicable to all of those.
+####> are applicable to all of those files.
```

Comment:
This change should be broken out into another commit.  It's causing a lot of noise.